install-dev/data/xml/address_format.xml

```diff
@@ -1054,16 +1054,6 @@ address1
 address2
 postcode city
 Country:name
-phone</format>
```

Comment:
The f*** is this?!!

Reply:
Some appended format for all countries that need to be split every 9 lines?! Or every `Country:name` instance?

Reply:
I remove an address format. Normal ;)